src/tests/ftest/pytorch/checkpoint.py

```diff
@@ -0,0 +1,100 @@
+"""
+  (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+  (C) Copyright 2025 Google LLC
+  (C) Copyright 2025 Enakta Labs Ltd
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import os
+import random
+import uuid
+
+from apricot import TestWithServers
+from pydaos.torch import Checkpoint
+
+
+class PytorchCheckpointTest(TestWithServers):
+    """Test Pytorch Checkpoint interface
+
+    :avocado: recursive
+    """
+
+    def test_checkpoint_no_chunking(self):
+        """Test Pytorch Checkpoint interface without chunking
+
+        Test Description: Ensure that single or multiple writes are read back correctly
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=vm
+        :avocado: tags=pytorch
+        :avocado: tags=PytorchCheckpointTest,test_checkpoint_no_chunking
+        """
+        pool = self.get_pool()
+        container = self.get_container(pool)
+
+        min_size = self.params.get("min_size", "/run/checkpoint_no_chunking/*", 1)
+        max_size = self.params.get("max_size", "/run/checkpoint_no_chunking/*", 4 * 1024 * 1024)
+        num_writes = self.params.get("writes", "/run/checkpoint_no_chunking/*", 7)
+
+        writes = []
+        for _ in range(num_writes):
+            writes.append(os.urandom(random.randint(min_size, max_size)))
```

Comment:
Python bandit. We have a wrapper for this one since it's common :)
```suggestion
            writes.append(os.urandom(self.random.randint(min_size, max_size)))
```